### PR TITLE
When clicking any element, seek its playback tick

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Not just "playable" elements, i.e. notes, chords...

Resolves: #10732
Resolves: #11493

### CODE REVIEW

**Comment by cbjeukendrup** [comment]
@DmitryArefiev Oops! Should be better in the next build :)

**Comment by DmitryArefiev** [comment]
> @DmitryArefiev Oops! Should be better in the next build :)

Сan you rebase please? The fix looks good on Mac, but some strange things on Linux.. And the build Win doesn't start because of libsndfile issue.

Thanks!

**Comment by cbjeukendrup** [comment]
@DmitryArefiev Done!

**Comment by DmitryArefiev** [comment]
Tested #10732 on Win10, Mac12, LinuxMint 20.2 - ** Looks good now. FIXED**      

Other related issue I will log separately

File: src/notation/inotationplayback.h [hunk 1]
```diff
@@ -56,6 +56,7 @@ class INotationPlayback
 
     virtual RectF playbackCursorRectByTick(midi::tick_t tick) const = 0;
 
+    virtual RetVal<midi::tick_t> playPositionTickByRawTick(midi::tick_t tick) const = 0;
```
Comment (RomanPudashkin) [inline src/notation/inotationplayback.h:59]: The purpose of this method is quite unclear. What is the difference between "regular" and "raw" ticks?

Comment (cbjeukendrup) [reply src/notation/inotationplayback.h:59]: It's basically a wrapper around `score()->repeatList().tick2utick(tick)`, which converts from "ticks without considering repeats" (i.e. just incrementing linearly from start to end of score) to "ticks with considering repeats" (i.e. unrolling all repeat sections before `tick`). Do you have a better idea for a name?

**Comment by DmitryArefiev** [comment]
> Other playback related issues I will log separately

Found related #11674

**Comment by DmitryArefiev** [comment]
@RomanPudashkin Please review :)

**Pull request merged**